ui/src/app/components/help-sidebar-icons.tsx

```diff
@@ -310,7 +313,10 @@ const displayIcon = (icon: IconConfig, props: DisplayIconProps) => {
         </a>
       ),
     ],
-    ['runtime', () => displayRuntimeIcon(icon, workspace.namespace)],
+    [
+      'runtimeConfig',
```

Comment:
the action taken is to show the runtime **config** panel, and the icon indicates the runtime **status**